### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by orionstardust**

### CODE REVIEW

File: contracts/interfaces/IERC173.sol [hunk 1]
```diff
@@ -5,12 +5,6 @@ pragma solidity ^0.8.0;
 ///  Note: the ERC-165 identifier for this interface is 0x7f5828d0
 /* is ERC165 */
 interface IERC173 {
-    /// @dev This emits when ownership of a contract changes.
-    event OwnershipTransferred(
```
Comment (cinnabarhorse) [inline contracts/interfaces/IERC173.sol:9]: This event isn't needed?

File: contracts/interfaces/IDiamondCut.sol [hunk 2]
```diff
@@ -33,6 +33,4 @@ interface IDiamondCut {
         bytes calldata _calldata
     )
         external;
-
-    event DiamondCut(FacetCut[] _diamondCut, address _init, bytes _calldata);
```
Comment (cinnabarhorse) [inline contracts/interfaces/IDiamondCut.sol:37]: This event isn't needed?

File: contracts/interfaces/IERC173.sol (hunk 1 above)
Comment (orionstardust) [reply contracts/interfaces/IERC173.sol:9]: @cinnabarhorse 
Yes, this function is not needed for this interface. i checked other diamonds.

File: contracts/interfaces/IDiamondCut.sol (hunk 2 above)
Comment (orionstardust) [reply contracts/interfaces/IDiamondCut.sol:37]: @cinnabarhorse
Yes, this function is not needed for this interface. i checked other diamonds.

**Pull request merged**